<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Dheenathayalans-MacBook-Air.local" failures="3" tests="53" name="runner.testRunnerTest" time="15.959" errors="0" timestamp="2025-02-12T10:26:07 EST" skipped="0">
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.198"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.135"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.215"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="3.890"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.206"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.224"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.194"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.205"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.204"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.205"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.124"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.230"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.209"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.223"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.221"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.207"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.191"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.222"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.332"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.217"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.207"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.217"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.212"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.282"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.202"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.208"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.204"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.287"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.207"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.172"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.223"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.280"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.373"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.256"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.213"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.321"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.307"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.211"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.392"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.217"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.305"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.243"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.101"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.340">
    <failure message="The following asserts failed:
	Status Code Validation Failed! expected [201] but found [500]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Status Code Validation Failed! expected [201] but found [500]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at utils.apiHelper.validateStatusCode(apiHelper.java:263)
at stepDefinitions.createUserStepDefinition.admin_receives_status_for_create_user_mandatory(createUserStepDefinition.java:53)
at ✽.Admin receives "201" "Created" Status for create user mandatory(file:///Users/mathumathibalakrishnan/eclipse/Mathumathi_2024/2025_Team04_RestAssured_BDD/src/test/resources/features/02_createUser.feature:14)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.204"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.197">
    <failure message="The following asserts failed:
	Status Code Validation Failed! expected [400] but found [409]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Status Code Validation Failed! expected [400] but found [409]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at utils.apiHelper.validateStatusCode(apiHelper.java:263)
at stepDefinitions.createUserStepDefinition.admin_receives_status_with_error_message(createUserStepDefinition.java:273)
at ✽.Admin receives "400" "Bad Request" Status with error message(file:///Users/mathumathibalakrishnan/eclipse/Mathumathi_2024/2025_Team04_RestAssured_BDD/src/test/resources/features/02_createUser.feature:54)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.376"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.113"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.210"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.198"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.513">
    <failure message="The following asserts failed:
	Status Code Validation Failed! expected [400] but found [201]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Status Code Validation Failed! expected [400] but found [201]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at utils.apiHelper.validateStatusCode(apiHelper.java:263)
at stepDefinitions.createUserStepDefinition.admin_receives_status_with_error_message(createUserStepDefinition.java:273)
at ✽.Admin receives "400" "Bad Request" Status with error message(file:///Users/mathumathibalakrishnan/eclipse/Mathumathi_2024/2025_Team04_RestAssured_BDD/src/test/resources/features/02_createUser.feature:149)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.214"/>
  <testcase classname="runner.testRunnerTest" name="runScenario" time="0.102"/>
</testsuite> <!-- runner.testRunnerTest -->
